src/hashmap/hashmap.ts

```diff
@@ -0,0 +1,39 @@
+import BinarySearchTree from '../tree/BST/binarySearchTree';
+
+const ARRAY_SIZE = 10;
+
+export default class HashMap<T> {
```

Comment:
T=any, so by default if not specified by the user the hashmap could handle any values